.circleci/config.yml

```diff
@@ -15,6 +15,23 @@ executors:
     resource_class: arm.medium
 
 jobs:
+  push-multi-arch:
+    machine:
+      image: ubuntu-2204:2022.07.1
+    steps:
+      - checkout
+      - attach_workspace:
+          at: .
+      - run:
+          name: Login
```

Comment:
I personally avoid those name: xxx when I use a simple command that is clear enough.
Imagine writing a script and adding a comment before each line ... that would be too much.

Reply:
I agree with you, but in this case I was copying a preexisting pattern used elsewhere in this build config. I suppose one benefit is that it separates a login failure vs. an actual build failure in the CircleCI interface.